node/src/db_config/config_dao_null.rs

```diff
@@ -283,4 +295,41 @@ mod tests {
 
         assert_eq!(value_pairs, expected_pairs);
     }
+
+    #[test]
+    fn set_works_simple() {
+        let subject = ConfigDaoNull::default();
+
+        subject.set("param1", Some("value1".to_string())).unwrap();
+        subject.set("schema_version", None).unwrap();
+        subject
+            .set("schema_version", Some("456".to_string()))
+            .unwrap();
+
+        let subject_data_sorted = subject.data.iter().sorted().collect::<Vec<(_, _)>>();
+        let comparison_subject_data_sorted = subject.data.iter().sorted().collect::<Vec<(_, _)>>();
+        assert_eq!(subject_data_sorted, comparison_subject_data_sorted)
+    }
+
+    #[test]
+    fn set_through_provided_transaction_works_simple() {
+        let subject = ConfigDaoNull::default();
+        let mut txn = TransactionWrapperMock::new();
+
+        subject
+            .set_through_provided_transaction(&mut txn, "param1", Some("value1".to_string()))
+            .unwrap();
+        subject
+            .set_through_provided_transaction(&mut txn, "schema_version", None)
+            .unwrap();
+        subject
+            .set_through_provided_transaction(&mut txn, "schema_version", Some("456".to_string()))
+            .unwrap();
+
+        let subject_data_sorted = subject.data.iter().sorted().collect::<Vec<(_, _)>>();
```

Comment:
This test has the same weakness as the test before this.

Reply:
Thank you!